### PR TITLE
Release on pypi

### PR DESCRIPTION
**Pull request by benrobby**
releases this library on pypi. To release a new version to pypi, do the following: 
- bump the version in setup.cfg
- go to https://github.com/hpi-epic/manm-cs/releases and create a new release (and a git tag for it) with the same name as the version
- a github action will run on release and first test this deployment on testPyPi and the publish to pypi

I still need to try out the last two steps when this PR is merged.

### CODE REVIEW

**Comment by benrobby** [comment]
I'm not 100% sure that everything works correctly with the changed imports because the tests don't cover much. It would be great if you could check that the experiments still work

**Comment by ChristopherSchmidt89** [comment]
Suggestion: let's move the experiments folder into a separate branch. It is not relevant for the actual manm-cs, but rather an example on how to use it in the interplay with mpcsl. What do you think?

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,33 @@
+[metadata]
+name = manm_cs
+version = 0.0.5
+author = Johannes Huegle, Christopher Hagedorn
```
Comment (ChristopherSchmidt89) [inline setup.cfg:4]: would also add the students who contributed to the package Lukas Boehme, Mats Poerschke, Jonas Umland

File: README.md [hunk 2]
```diff
@@ -42,24 +52,37 @@ py -m venv env
 .\env\Scripts\activate
 ```
 
-After the creation of a new virtual enviroment, we can install the project dependencies defined in [requirements.txt](requirements.txt) for both platforms.
+After the creation of a new virtual enviroment, we can install the project dependencies defined in [setup.cfg](setup.cfg) for both platforms.
```
Comment (ChristopherSchmidt89) [inline README.md:55]: correct typo in environment

**Comment by JohannesHuegle** [comment]
@ChristopherSchmidt89 @benrobby a separate branch to present the interplay with MPCSL is a good idea. In particular, as most examples are outdated.

**Pull request merged**